templates/scenarios/sso-tab-js/teamsfx/app.local.yml

```diff
@@ -20,10 +20,10 @@ registerApp:
     # TEAMS_APP_ID: the id of Teams app
 
 configureApp:
-  - uses: env/generate
+  - uses: env/generate # Generate env to .env file
     with:
       envs:
-        TAB_DOMAIN: localhost # Generate env to .env file
+        TAB_DOMAIN: localhost
```

Comment:
Add port.

Reply:
It is used in aad and do not need port.

Reply:
Due to https://github.com/OfficeDev/TeamsFx/pull/6417?

Reply:
Yes.